pandas/tests/scalar/test_nat.py

```diff
@@ -156,6 +158,44 @@ def test_NaT_methods():
     assert NaT.isoformat() == 'NaT'
 
 
+def test_NaT_docstrings():
+    nat_names = dir(NaT)
+
+    ts_names = dir(Timestamp)
+    ts_missing = [x for x in ts_names if x not in nat_names and
+                  not x.startswith('_')]
+    ts_missing.sort()
+    ts_expected = ['freqstr', 'normalize', 'offset',
+                   'to_julian_date', 'to_period', 'tz']
+    assert ts_missing == ts_expected
+
```

Comment:
give a comment before each 'section' of the test so a future reader gets what you are testing